components/src/core/Hero/Hero.tsx

```diff
@@ -37,66 +43,74 @@ export type HeroProps = HTMLAttributes<HTMLDivElement> & {
-            width: (props: HeroProps): number | string =>
-                typeof props.iconSize === 'number' ? Math.max(44, props.iconSize) : props.iconSize,
-            backgroundColor: (props: HeroProps): string => props.iconBackgroundColor,
-        },
-        values: {
-            display: 'flex',
-            alignItems: 'center',
-            color: theme.palette.text.primary,
-            lineHeight: 1.2,
-            maxWidth: '100%',
-            overflow: 'hidden',
-            fontSize: '1.25rem',
-        },
-        label: {
-            fontSize: 'inherit',
-            lineHeight: 1.2,
-            letterSpacing: 0,
-            fontWeight: 600,
-            width: '100%',
-            textAlign: 'center',
-            overflow: 'hidden',
-            textOverflow: 'ellipsis',
-            whiteSpace: 'nowrap',
-        },
-    })
-);
+const Root = styled(Box, { name: 'hero', slot: 'root' })<Pick<HeroProps, 'onClick'>>(({ onClick, theme }) => ({
+    fontSize: '1rem',
+    display: 'flex',
+    flexDirection: 'column',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+    flex: '1 1 0px',
+    overflow: 'hidden',
+    color: theme.palette.text.primary,
+    padding: `1rem ${theme.spacing()}`,
+    cursor: onClick ? 'pointer' : 'inherit',
+}));
+
+const Icon = styled(Box, {
+    name: 'hero',
+    slot: 'icon',
+})<Pick<HeroProps, 'iconSize' | 'iconBackgroundColor'>>(({ iconSize, iconBackgroundColor, theme }) => ({
+    lineHeight: 1,
+    color: theme.palette.text.secondary,
+    marginBottom: '.5rem',
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    textAlign: 'center',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+    padding: `.25rem ${theme.spacing(0.5)}`,
+    borderRadius: '50%',
+    fontSize: typeof iconSize === 'number' ? normalizeIconSize(iconSize) : iconSize,
+    height: typeof iconSize === 'number' ? Math.max(44, iconSize) : iconSize,
+    width: typeof iconSize === 'number' ? Math.max(44, iconSize) : iconSize,
+    backgroundColor: iconBackgroundColor,
+}));
+
+const ChannelValueContainer = styled(Box, {
+    name: 'hero',
+    slot: 'values',
+})(({ theme }) => ({
+    display: 'flex',
+    alignItems: 'center',
+    color: theme.palette.text.primary,
+    lineHeight: 1.2,
+    maxWidth: '100%',
+    overflow: 'hidden',
+    fontSize: '1.25rem',
+}));
+
+const Label = styled(Typography, {
+    name: 'hero',
+    slot: 'label',
+})(() => ({
+    fontSize: 'inherit',
+    lineHeight: 1.2,
+    letterSpacing: 0,
+    fontWeight: 600,
+    width: '100%',
+    textAlign: 'center',
+    overflow: 'hidden',
+    textOverflow: 'ellipsis',
+    whiteSpace: 'nowrap',
+}));
 
 const HeroRender: React.ForwardRefRenderFunction<unknown, HeroProps> = (props: HeroProps, ref: any) => {
-    const defaultClasses = useStyles(props);
+    const ownerState = {
+        ...props,
+    };
+    const defaultClasses = useUtilityClasses(ownerState);
```

Comment:
we can pass props directly instead of storing it in this ownerState object
```suggestion

    const defaultClasses = useUtilityClasses(props);
```